static/js/metrics-explorer.js

```diff
@@ -1123,26 +1121,26 @@ async function convertDataForChart(data) {
 
 async function getMetricNames() {
     const data = {
-      start: filterStartDate,
-      end: filterEndDate,
+        start: filterStartDate,
+        end: filterEndDate,
     };
     const res = await $.ajax({
-      method: "post",
-      url: "metrics-explorer/api/v1/metric_names",
-      headers: {
-        "Content-Type": "application/json; charset=utf-8",
-        Accept: "*/*",
-      },
-      crossDomain: true,
-      dataType: "json",
-      data: JSON.stringify(data),
+        method: "post",
+        url: "http://playground.siglens.com:5122/metrics-explorer/api/v1/metric_names",
```

Comment:
you should not hard code the server url here

Reply:
I just had it hardcoded for the purpose of demoing my changes to you, because the `localhost:5122/metrics-explorer/api/vi/metric_names` url was not returning any data and was just returning an empty array. I can remove this hardcoded string to original value after the demo if you'd like a demo from me.